actionpack/lib/action_dispatch/http/content_security_policy.rb

```diff
@@ -175,7 +175,9 @@ def plugin_types(*types)
     end
 
     def report_uri(uri)
-      @directives["report-uri"] = [uri]
+      unless uri.nil?
+        @directives["report-uri"] = [uri]
+      end
```

Comment:
Do we care about unsetting this? Something like:
```
@directives["report-uri"] = if uri.nil?
    nil
  else
    [uri]
  end
```

Reply:
I've just scrolled up (es, after opening the PR 🙄 ) and seen that there are a couple of other directives already doing, like so.

```rb
def block_all_mixed_content(enabled = true)
  if enabled
    @directives["block-all-mixed-content"] = true
  else
    @directives.delete("block-all-mixed-content")
  end
end
```

I might swap to this and see where that gets me for consistency.